lib/simplifyExpression/collectAndCombineSearch/multiplyLikeTerms.js

```diff
@@ -37,9 +37,36 @@ function multiplyLikeTerms(node, polynomialOnly=false) {
     return status;
   }
 
+  status = multiplyNthRoots(node);
+  if (status.hasChanged()) {
+    return status;
+  }
+
   return Node.Status.noChange(node);
 }
 
+function multiplyNthRoots(node) {
+  if (!checks.canMultiplyLikeTermsNthRoots(node)){
+    return Node.Status.noChange(node);
+  }
+
+  let newNode = clone(node);
+
+  // Array of bases of all the nthRoot terms being multiplied
+  const radicands = node.args.map(term => term.args[0]);
+
+  // Multiply them
+  const newRadicandNode = Node.Creator.operator('*', radicands);
+
+  // All the args at this point have the same index,
```

Comment:
I think we've been using a different name for this than index (forget off the top of my head, but the root code will definitely use it - it might just be 'root' i.e why you're calling it rootNode :p)

is index the official name for the "degree" of the root?